### DISCUSSION TITLE
Blacklist of commands for portals

### DISCUSSION THREAD

**Discussion by Pinball3D** [Zrips/CMI#6286] [Q&A]
Hello. I am running a survival smp. for one of the ranks players will be able to make portals through cmi. the only problem is that you can make portals that will have the stop command activate on enter. Is there any way to add a blacklist of commands?

**Comment by mrfloris** [comment]
How do you prevent a player from making the whole world one portal and ruining the server (sorry for off topic question)

Portals is really an admin/owner command, not something for everybody to use.

There aren't really any options or permissions that i can tell that could help you limit what you can do. 
A blacklist? I rather see a whitelist, where you can allow a few commands, rather than add 500+ commands a server could potentially have. 

But again, I really personally recommend against granting players access to the CMI portal creation features. 

<img width="943" alt="Screenshot 2021-11-27 at 10 51 51" src="https://user-images.githubusercontent.com/28841349/143676420-ff7cc4de-4ca9-43b1-8f6b-704c51c9d435.png">
